src/api/app/models/bs_request.rb

```diff
@@ -107,6 +108,21 @@ def check_creator
     errors.add(:creator, "Login #{user.login} is not an active user")
   end
 
+  def double_actions
```

Comment:
my personal opinion is that adding code in the header if it is just used in one place causes more distraction then it helps. but up to you ..